app/views/home/about.html.haml

```diff
@@ -0,0 +1,50 @@
+
+        %div.mt-5
+            %div.mb-3{style:"display: flex;justify-content: center;align-items: center;"}   
+                %h3 Founder
+        .row.justify-content-md-center.mt-0
+            .col-md-3
+                .card.card-orange-blue
+                    %div.mt-1.mb-5{style:"border-radius: 50%;align-items: flex-start;width: 6rem;height: 6rem;background-color: white;margin: 0 auto;margin-top: 5%"}         
+                    %div.mt-5{style:"align-items: flex-end;margin: 0 auto"}
+                        %img{alt:"LinkedIN image", src: asset_path('LinkedIn.png')}
+                        %img{alt:"Github image", src: asset_path('Github.png')}
+        %div.mt-4
+            %div{style:"display: flex;justify-content: center;align-items: center;"}   
+                %h3 Contributors
+        .row.justify-content-md-center.mt-4
+            .col-md-3
+                .card.card-orange-blue
+                    %div.mt-1.mb-5{style:"border-radius: 50%;align-items: flex-start;width: 6rem;height: 6rem;background-color: white;margin: 0 auto;margin-top: 5%"} 
+                    %div.mt-5{style:"align-items: flex-end;margin: 0 auto"}
+                        %img{alt:"LinkedIN image", src: asset_path('LinkedIn.png')}
+                        %img{alt:"Github image", src: asset_path('Github.png')}
+            .col-md-3
+                .card.card-orange-blue
+                    %div.mt-1.mb-5{style:"border-radius: 50%;align-items: flex-start;width: 6rem;height: 6rem;background-color: white;margin: 0 auto;margin-top: 5%"}
+                    %div.mt-5{style:"align-items: flex-end;margin: 0 auto"}
+                        %img{alt:"LinkedIN image", src: asset_path('LinkedIn.png')}
+                        %img{alt:"Github image", src: asset_path('Github.png')}
+            .col-md-3
+                .card.card-orange-blue
+                    %div.mt-1.mb-5{style:"border-radius: 50%;align-items: flex-start;width: 6rem;height: 6rem;background-color: white;margin: 0 auto;margin-top: 5%"}
+                    %div.mt-5{style:"align-items: flex-end;margin: 0 auto"}
+                        %img{alt:"LinkedIN image", src: asset_path('LinkedIn.png')}
+                        %img{alt:"Github image", src: asset_path('Github.png')}
+            .col-md-3
+                .card.card-orange-blue
+                    %div.mt-1.mb-5{style:"border-radius: 50%;align-items: flex-start;width: 6rem;height: 6rem;background-color: white;margin: 0 auto;margin-top: 5%"}
+                    %div.mt-5{style:"align-items: flex-end;margin: 0 auto"}
+                        %img{alt:"LinkedIN image", src: asset_path('LinkedIn.png')}
+                        %img{alt:"Github image", src: asset_path('Github.png')}
```

Comment:
Could you fix indentation to 2 spaces i.e 1 tab - it's currently 2 tabs 4 spaces.